src/fu-tool.c

```diff
@@ -3545,11 +3546,12 @@ main(int argc, char *argv[])
 	fu_util_cmd_array_sort(cmd_array);
 
 	/* non-TTY consoles cannot answer questions */
-	priv->interactive = isatty(fileno(stdout)) != 0;
```

Comment:
This also goes on fu-util right?

Reply:
Not sure I understand the question, sorry.

Reply:
Don't we also have an interactive check there too?

Reply:
Ahh! Indeed we do. Fixed.